README.md

```diff
@@ -73,7 +73,9 @@ Some of the rules supported by the makefile:
 | kustomize | install the kustomize tool, used by other commands |
 | docker-build | build registry operator docker image |
 | docker-push | push registry operator docker image |
-| docker-buildx | build & push registry operator docker image for all supported architectures \(**does not work with podman**\) |
+| docker-buildx | build & push registry operator docker image for all supported architectures |
+| podman-build | build registry operator podman image |
+| podman-push | push registry operator podman image |
```

Comment:
I would phase it "build registry operator container image using `<engine>`"

```suggestion
| podman-build | build registry operator container image using podman |
| podman-push | push registry operator container image using podman |
```

Reply:
@michael-valdron updated the wording in the most recent commit to reflect the engine being used for building/pushing the images